### PR TITLE
refactored cyclops keyboard project

### PR DESCRIPTION
**Pull request by westfoxtrot**
## Description
Moved the files into a subfolder to allow consistency in preparation for additional commits to come in the future.

made some small adjustments to the configuration and keymap

## Types of changes
- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

N/A

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/westfoxtrot/cyclops/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# cyclops
+
+![cyclops](https://github.com/westfoxtrot/Cyclops)
+
+66% Alps Keyboard in ISO layout.
+
+Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)
```
Comment (drashna) [inline keyboards/westfoxtrot/cyclops/readme.md]:
```suggestion
 Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)  
```

File: keyboards/westfoxtrot/cyclops/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Hardware Supported: Cyclops (AEKISO66) PCB
```
Comment (drashna) [inline keyboards/westfoxtrot/cyclops/readme.md]:
```suggestion
 Hardware Supported: Cyclops (AEKISO66) PCB  
```

File: keyboards/westfoxtrot/cyclops/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Hardware Availability: Files released here (https://github.com/westfoxtrot/Cyclops)
```
Comment (drashna) [inline keyboards/westfoxtrot/cyclops/readme.md]:
```suggestion
 Hardware Availability: Files released here (https://github.com/westfoxtrot/Cyclops)  
```

File: keyboards/westfoxtrot/cyclops/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/westfoxtrot/cyclops/keymaps/default/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

**Comment by drashna** [comment]
Also, since this appears to be a very limited run, or personal...  Could you move this board to the `handwired` folder?

**Comment by westfoxtrot** [comment]
> 
> 
> Also, since this appears to be a very limited run, or personal... Could you move this board to the `handwired` folder?

I can do but I'm just updating files that were previously commited to keyboards/cyclops as you see in files changed. What's the guidance or limit for something being commited directly to keyboards versus to handwire?

**Comment by mechmerlin** [comment]
> Also, since this appears to be a very limited run, or personal... Could you move this board to the `handwired` folder?

I would argue that since the Cyclops is an actual PCB, not a prototype, and has sold ~20, it can have its own directory rather than being in the handwired one.

**Comment by westfoxtrot** [comment]
Thanks! And they're some plans to make an ANSI one but the layout is slightly harder to cover with just an AEK set so we need to plan it out a bit more. On 28 Dec 2018 20:09, MechMerlin <notifications@github.com> wrote:@mechmerlin approved this pull request.

I want this in ANSI =).

**Pull request merged**